DEVELOPMENT.md

```diff
@@ -43,43 +23,70 @@ Poetry manages virtualenvs as well. Typically, on a project that uses virtualenv
 poetry shell
 ```
 
-You shouldn't have to actually do that unless you want to run some of the things that are pip installed directly. One use case would be manually running `pytest` on a single test file rather than running them all with `make test`.
-
-## Managing Dependencies
+You shouldn't have to actually do that unless you want to run some of the executables things that are installed from pypi directly. One use case would be manually running `pytest` on a single test file rather than running them all with `make test`.
```

Comment:
```suggestion
You shouldn't have to actually do that unless you want to run some of the executables that are installed from pypi directly. One use case would be manually running `pytest` on a single test file rather than running them all with `make test`.
```